File: js/task2/S2_ARR_02.js
function unique(values) {
  const result = [];

  for (const value of values) {
      if (!result.includes(value)) {
          result.push(value);
      }
  }

  return result;
}

console.log(unique([1, 2, 3, 2, 1]));       
console.log(unique(["a", "b", "a", "c"]));   
console.log(unique([1, "1", 1, "1"]));       
console.log(unique([]));                     
console.log(unique([1, 2, 3]));

File: python/task2/S2_ARR_02.py
def unique(values):
  result = []
  for value in values:
      if value not in result:
          result.append(value)
  return result

print(unique([1, 2, 3, 2, 1]))
print(unique(["a", "b", "a", "c"]))
print(unique([1, "1", 1, "1"]))
print(unique([]))
print(unique([1, 2, 3]))
